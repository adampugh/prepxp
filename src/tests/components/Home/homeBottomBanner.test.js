import React from "react";
import { shallow } from "enzyme";

import HomeBottomBanner from "../../../components/Home/homeBottomBanner";

describe("<HomeBottomBanner />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<HomeBottomBanner />);
        expect(wrapper).toMatchSnapshot();
    });
});