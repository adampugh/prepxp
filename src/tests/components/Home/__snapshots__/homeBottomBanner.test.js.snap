// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeBottomBanner /> should match snapshot 1`] = `
<section
  className="bottomBanner"
>
  <div
    className="container"
  >
    <h1>
      Land your dream job
    </h1>
    <p>
      Sign up to PrepXP today to better your career and get the job you deserve! Pass the interview by gaining experience and preparing for success in all future job interviews.
    </p>
    <Link
      replace={false}
      to="/prepxp/login"
    >
      <button
        className="btn"
      >
        Start now
      </button>
    </Link>
    <img
      alt="policeman, business man and designer"
      src="bottomBanner.png"
    />
  </div>
</section>
`;
